src/components/Navbar.css

```diff
@@ -1,20 +1,28 @@
 .navbar {
   background: rgb(233, 111, 37);
-  height: 80px;
   display: flex;
   justify-content: center;
   align-items: center;
   font-size: 1.2rem;
-  position: sticky;
-  top: -80px;
   z-index: 999;
   font-family: 'Montserrat', sans-serif;
   font-weight: 600;
   transition: top 1s;
+  overflow: hidden;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  transition: height 500ms ease-in-out, opacity 400ms ease-in-out;
```

Comment:
`Position: fixed` and `top:0` together will keep it fixed at the top. Because the position becomes fixed, you need to make the width the length of the page (100%).

Added transitions to make it a smoother transition.